src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/DefaultHtmlGenerator.cs

```diff
@@ -802,7 +802,7 @@ public virtual IHtmlContent GenerateAntiforgery(ViewContext viewContext)
             {
                 foreach (var modelError in modelState.Errors)
                 {
-                    var errorText = ValidationHelpers.GetUserErrorMessageOrDefault(modelError, modelState: null);
+                    var errorText = ValidationHelpers.GetUserErrorMessageOrDefault(modelError, modelState);
```

Comment:
This is a change from validation summaries provided in MVC 5 and earlier. It's been consistent for validation messages (at field level) to include default "is invalid" message but to hide such messages in validation summaries. What I expected to fix #3043 was another special case in `TryAddModelError(... Exception ...)` for the  `Exception` type thrown in that scenario (`OverflowException`?), similar to `FormatException` handling but with a different message e.g. `Value '{0}' is out of range for '{1}'.`. @Eilon your thoughts?

If we decide to use current approach, should remove the `modelState == null` special case in `GetUserErrorMessageOrDefault()`. Also need an integration test demonstrating new behaviour for exact #3043 scenario (attempt to bind a negative value to a `uint` property).